aldryn_github_stats/tasks.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from celery.schedules import schedule
+from celery.task import periodic_task
+from celery.utils.log import get_task_logger
+
+from django.conf import settings
+
+from .models import GitHubStatsRepository
+
+logger = get_task_logger(__name__)
+
+CACHE_DURATION = getattr(settings, "ALDRYN_GITHUB_STATS_CACHE_DURATION", 3600)
+
+
+@periodic_task(run_every=(schedule(run_every=CACHE_DURATION)))
```

Comment:
You might've noticed later than in GitHubStatsRepository.get_repo() I handle this "the other" way, I cache things 2X the ALDRYN_GITHUB_STATS_CACHE_DURATION when its a Celery fetch. This way we also have a buffer in case of network delays or whatever.